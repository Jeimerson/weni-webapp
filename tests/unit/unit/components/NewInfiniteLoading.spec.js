import { shallowMount } from '@vue/test-utils';
import NewInfiniteLoading from '@/components/NewInfiniteLoading.vue';

describe('NewInfiniteLoading.vue', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(NewInfiniteLoading, {
      mocks: {
        $t: () => '',
      },
    });
  });

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
