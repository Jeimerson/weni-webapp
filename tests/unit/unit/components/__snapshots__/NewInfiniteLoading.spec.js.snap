// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewInfiniteLoading.vue renders correctly 1`] = `undefined`;
